imports/ui/styles/_cards.scss

```diff
@@ -0,0 +1,8 @@
+.card-deck .card {
```

Comment:
Prefer to lean in on Bootstrap's utility methods instead of making new ones - https://getbootstrap.com/docs/4.0/utilities/spacing/

If you need the MatchCard component to have a margin bottom, just add a class to the `<Card>` component. Try using `.mb-2`. 

Similarly, you shouldn't need to set a new `small` CSS rule, try using `<p><small> ... </small></p>` or possibly the `.small` class. 
https://getbootstrap.com/docs/4.0/content/typography/#inline-text-elements

Reply:
Nice, I didn't know about the Bootstrap's spacing method! I removed the .card-deck .card. The <small> I defined I was trying to override, since the current `small` rule isn't small enough. 

```<Card.Footer><small className="text-muted">{ timezone }</small></Card.Footer>```

![image](https://user-images.githubusercontent.com/4512699/55379377-c2867300-54d1-11e9-8d44-43a22be6f962.png)

Reply:
Hm, granted the font-weight and muted classing, I'm afraid it might actually be too small and too light for it to be accessible / readable:

![image](https://user-images.githubusercontent.com/3891869/55380716-e3e95e00-54d5-11e9-906d-17fb4235df37.png)

imo, it's not really worth overriding :\